hail/python/test/hail/methods/test_impex.py

```diff
@@ -548,19 +546,20 @@ def concat_files(outpath, inpaths):
         mt = hl.import_vcf(resource('sample.vcf'), min_partitions=4)
         f = new_temp_file(extension='vcf.bgz')
         hl.export_vcf(mt, f, parallel='separate_header')
-        shard_paths = glob.glob(f + "/*.bgz")
+        shard_paths = fs.ls(f + "/*.bgz")
```

Comment:
I don't think this will work against non-hadoop FileSystems. I think the easy thing to do is just list the entire directory and filter to files with the bgz suffix. Not the smartest thing, but will definitely work.